src/style.css

```diff
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+input {
+  font-family: inherit;
+  padding: 0;
+}
+
+.search-bar {
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin: 20px 0;
+  position: relative;
+  cursor: pointer;
+}
+
+.search-bar:last-of-type {
```

Comment:
What if we will change position of inputs later ?
Add specific class, like `search-bar--small`